packages/components/NetworkSelector/NetworkSelectorMenu.tsx

```diff
@@ -75,24 +75,23 @@ export const NetworkSelectorMenu: FC<{
   };
 
   return (
-    <LegacyTertiaryBox
-      width={172}
-      noBrokenCorners
-      style={style}
-      mainContainerStyle={[
-        {
-          paddingHorizontal: layout.spacing_x2,
-          paddingTop: layout.spacing_x2,
-          backgroundColor: neutral17,
-          alignItems: "flex-start",
-        },
-        isMobile && {
-          borderTopWidth: 0,
-          borderRightWidth: 0,
-          borderTopRightRadius: 0,
-          borderBottomRightRadius: 0,
-        },
-      ]}
+    <TertiaryBox
+      style={{
+        ...[style],
+        paddingHorizontal: layout.spacing_x2,
+        paddingTop: layout.spacing_x2,
+        backgroundColor: neutral17,
+        alignItems: "flex-start",
+        ...(isMobile
+          ? {
+              borderTopWidth: 0,
+              borderRightWidth: 0,
+              borderTopRightRadius: 0,
+              borderBottomRightRadius: 0,
+            }
+          : {}),
```

Comment:
I think we can make that more readable, you could do like that:
```
style={[style, {blabla}, isMobile && {blabla} ]}
```

Reply:
done.